lib/launcher.rb

```diff
@@ -2,6 +2,7 @@
 require 'proxy/sd_notify'
 require 'proxy/settings'
 require 'proxy/signal_handler'
+require 'proxy/log_buffer/trace_decorator'
```

Comment:
did you forget to `git add` this? ;)

Reply:
Probably, let me check. It happened in the past :-)